ramls/schema/loan-policy-id.json

```diff
@@ -7,6 +7,13 @@
     "loanPolicyId": {
       "description": "ID of the loan policy determined by the circulation rules",
       "type": "string"
+    },
+    "conditions": {
```

Comment:
Are these the conditions that matched for the rule to be applied?

If so, I suggest the API explicitly state that in the name and description, as conditions could be ambiguous without that context.

Reply:
renamed as "appliedRuleConditions"